codec-http/src/main/java/io/netty/handler/codec/http/HttpContentCompressor.java

```diff
@@ -48,26 +50,25 @@
  */
 public class HttpContentCompressor extends HttpContentEncoder {
 
-    private final boolean supportsCompressionOptions;
     private final BrotliOptions brotliOptions;
     private final GzipOptions gzipOptions;
     private final DeflateOptions deflateOptions;
     private final ZstdOptions zstdOptions;
     private final SnappyOptions snappyOptions;
 
-    private final int compressionLevel;
-    private final int windowBits;
-    private final int memLevel;
     private final int contentSizeThreshold;
     private ChannelHandlerContext ctx;
     private final Map<String, CompressionEncoderFactory> factories;
 
     /**
-     * Creates a new handler with the default compression level (<tt>6</tt>),
-     * default window size (<tt>15</tt>) and default memory level (<tt>8</tt>).
+     * Creates a new handler with {@link StandardCompressionOptions#brotli()},
+     * {@link StandardCompressionOptions#zstd()}, {@link StandardCompressionOptions#snappy()},
+     * {@link StandardCompressionOptions#gzip()} and {@link StandardCompressionOptions#deflate()}.
      */
     public HttpContentCompressor() {
-        this(6);
+        this(0, StandardCompressionOptions.brotli(), StandardCompressionOptions.zstd(),
```

Comment:
Isn't it better to call `this((CompressionOptions[]) null)`

Reply:
I agree... Let me fix this.

Reply:
@violetagg https://github.com/netty/netty/pull/14519